ledger-service/http-json/src/it/scala/http/AbstractHttpServiceIntegrationTest.scala

```diff
@@ -468,7 +404,7 @@ trait AbstractHttpServiceIntegrationTestFuns extends StrictLogging {
   protected def initialAccountCreate(
       serviceUri: Uri,
       encoder: DomainJsonEncoder): Future[(StatusCode, JsValue)] = {
-    val command = accountCreateCommand(domain.Party("Alice"), "abc123")
+    val command = HttpServiceTestFixture.accountCreateCommand(domain.Party("Alice"), "abc123")
```

Comment:
In general I think `import HttpServiceTestFixture._` or importing only the symbols used would be cleaner than writing all the points of use in an old Java style.

Reply:
fixed in 9329063b55544277e9c0b133ceb9056d2f86158e